storage/entities/base_entity.js

```diff
@@ -204,6 +204,24 @@ class BaseEntity {
 		return this._getValueSetForObject(data, attributes);
 	}
 
+	/**
+	 * Load given SQL files to given entity
+	 *
+	 * @param {string} entityLabel - Namespace of the entity
+	 * @param {Object} sqlFiles - Object with SQL label as key and path to load as value
+	 * @return {Object}
+	 */
+	loadSQLFiles(entityLabel, sqlFiles) {
+		this.adapter.SQLs = this.adapter.SQLs || {};
+		this.adapter.SQLs[entityLabel] = this.adapter.SQLs[entityLabel] || {};
+		Object.keys(sqlFiles).forEach(key => {
+			if (!this.adapter.SQLs[entityLabel][key]) {
+				this.adapter.SQLs[entityLabel][key] = this.adapter.loadSQLFile(sqlFiles[key]);
+			}
+		});
+		return this.adapter.SQLs[entityLabel];
+	}
```

Comment:
With the following change, this function will become integral component for the storage layer. Please add some unit tests for it.